skills/bootstrap/SKILL.md

```
@@ -0,0 +1,332 @@
+
+`$ARGUMENTS` — опциональное описание проекта.
+
+Если пуст — скилл определит всё автоматически.
+
+---
+
+## Pipeline
+
+7 фаз. Отмечай каждую через TodoWrite.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)